src/nsrt_learning.py

```diff
@@ -206,8 +235,9 @@ def _update_segment_from_option_specs(segment: Segment,
     Modifies segment in place.
     """
     assert not CFG.do_option_learning, "TODO: implement option learning."
-    assert not segment.has_option()
-    segment.set_option_from_trajectory()
+    # Note: after option learning is implement, use:
+    #   segment.set_option_from_trajectory()
```

Comment:
This comment confuses me --- won't segment.set_option_from_trajectory crash if there are no options? The whole point of this _update_segment_from_option_specs method should be to add in those options...

Reply:
I actually feel that this whole `_update_segment_from_option_specs` method is useless in the non-option-learning code path. Could we just, in the top-level method, not run this method if CFG.do_option_learning is false?

Reply:
And now i'm even more confused about why segment.set_option_from_trajectory wouldn't just crash, if we're doing option learning

Reply:
you're right, that comment was wrong. I'll remove, and yes, we can skip the method entirely